### ISSUE TITLE
No callback is fired when video button is clicked

### ISSUE THREAD

**Issue by VictorUvarov** [peerwaya/flutter_call_kit#7]
- When receiving a voip call with CallKit in the background and video is selected you are given a screen that has a video button.
- When clicking that button, no callback is fired. Is this a missing feature?

**Comment by yukitaka13-1110** [comment]
@VictorUvarov 
Are you mean `performAnswerCallAction ` callback ?

**Comment by VictorUvarov** [comment]
No, after the call is answered from the background a screen with a video button occurs. Clicking the video button doesn't fire any callback. Like the screen show here [video screen](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse4.mm.bing.net%2Fth%3Fid%3DOIP.KTMka0KgbhJVgIKHxf3yRQHaEo%26pid%3DApi&f=1)

**Comment by peerwaya** [comment]
the video button does not answer the call,  call kit opens your app so you can render the video

**Comment by VictorUvarov** [comment]
It opens for you? I just prompts for face unlock then does nothing

**Comment by livotov** [comment]
@VictorUvarov did you find any solution for this?
Happening to me as well.

**Comment by VictorUvarov** [comment]
No solution. I'm using swift, not sure if that's a problem

**Comment by livotov** [comment]
Yep, I'm on swift too, it should not be a problem. Maybe just the bugs in original ObjC implementation here. 

Forked this repo and will try to complete missing things. The user activity continuation (when you start a call from the system call log) also not covered here and needs to be implemented.

**Comment by VictorUvarov** [comment]
Cool i'll keep an eye out if you get that implemented
